src/js/components/AppPageComponent.jsx

```diff
@@ -410,6 +411,14 @@ var AppPageComponent = React.createClass({
       );
     }
 
+    var deploymentTroublesDoc = (model.status !== AppStatus.RUNNING) ? (
+      <div className="doc-deployment-troubles">
+        <a href="https://confluence.criteois.com/x/FcLzFw" target="_blank">
```

Comment:
FAR: make it configurable from the outside to avoid leaking these links to public repos.